core/src/main/java/org/axonframework/config/DefaultSagaEventProcessorRegistry.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2010-2018. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.config;
+
+import org.axonframework.eventhandling.EventProcessor;
+import org.axonframework.eventhandling.MultiEventHandlerInvoker;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Default implementation of {@link EventProcessorRegistry}.
+ *
+ * @author Milan Savic
+ * @since 3.3
+ */
+class DefaultSagaEventProcessorRegistry implements EventProcessorRegistry {
```

Comment:
What makes this `EventProcessorRegistry` specific for Sagas? 

It seems that the `DefaultConfigurer` will default to the `DefaultSagaEventProcessorRegistry`, which to me suggested it will be used for the `SagaConfiguration` and the `EventHandlingConfiguration.
Having '...Saga...' in the class name thus seemed weird to me.

Reply:
We are going to lose this one.